### ISSUE TITLE
archive 2018 edition in branch

### ISSUE THREAD

**Issue by lachlandeer** [pp4rs/syllabus#5]
2018 edition of the syllabus should be archived in a branch

**Comment by lachlandeer** [comment]
2018 syllabus doesn't appear to be in repo ... need to go hunting to see what @pp4rs/instructors-2018-uzh did with their syllabus

**Comment by bergmul** [comment]
Compiled version is under is under https://pp4rs.github.io/2018-uzh/schedule/
**EDIT** Source is in 2018 website in https://github.com/pp4rs/2018-uzh/blob/master/docs/schedule.md

**Comment by lachlandeer** [comment]
@bergmul - any idea where the source code is?

**Comment by bergmul** [comment]
@lachlandeer see edit above

**EDIT** Seems we just copy and pasted the HTML code for the table from an online table generator.

**Comment by lachlandeer** [comment]
OK - no idea where the actual syllabus Rmd etc is? Or perhaps they didnt create one?

**Comment by bergmul** [comment]
Found it. It lived outside of the repo and was distributed via email. TBH I believe it would be better to use something else as a template.

This is a .md file but I added it here with a .txt ending for filetype restrictions. Feel free to add it to the place it belongs.

[2018-PP4RS-sylabus.txt](https://github.com/pp4rs/syllabus/files/3673573/2018-PP4RS-sylabus.txt)

**EDIT** I would also not vouch for its correctness. The website' schedule superseded it after the start of the course.
